src/field_comm_manager.c

```diff
@@ -73,20 +73,20 @@ static void sub_02059658 (void)
     return;
 }
 
-void FieldCommMan_Init (FieldSystem * fieldSys)
+void FieldCommMan_Init (FieldSystem * fieldSystem)
 {
     if (sFieldCommMan != NULL) {
         return;
     }
 
-    CommFieldCmd_Init((void *)fieldSys);
+    CommFieldCmd_Init((void *)fieldSystem);
```

Comment:
Is this cast necessary?

Reply:
nope, I'll start removing them as I see them lol